src/test/java/com/jakewharton/retrofit2/adapter/kotlin/coroutines/CancelTest.kt

```diff
@@ -36,7 +36,7 @@ class CancelTest {
     val deferred = adapter.adapt(call)
     call.completeWithException(IOException())
     assertFalse(call.isCanceled)
```

Comment:
The call is cancelled in coroutines 0.27.0+ because [`CompletedExceptionally` is considered a "cancelled" state](https://github.com/Kotlin/kotlinx.coroutines/blob/0.27.0/common/kotlinx-coroutines-core-common/src/JobSupport.kt#L181-L184)